demisto_sdk/commands/content_graph/objects/base_content.py

```diff
@@ -282,14 +281,15 @@ def from_path(
             )
             return None
 
-        model = content_type_to_model.get(content_item_parser.content_type)
+        model = CONTENT_TYPE_TO_MODEL.get(content_item_parser.content_type)
         logger.debug(f"Loading content item from path: {path} as {model}")
         if not model:
             logger.error(f"Could not parse content item from path: {path}")
             return None
         try:
-            obj = model.from_orm(content_item_parser)
-            obj.git_status = git_status
+            obj = model.from_orm(content_item_parser)  # type: ignore
+            if obj:
+                obj.git_status = git_status
```

Comment:
same note as here: https://github.com/demisto/demisto-sdk/pull/3761#discussion_r1385620008

Reply:
We can't really move this field to the content parser, as we don't have this data during parsing in the current implementation.
If we want to add it to the parser (and to the graph), it will require us to call `git_util.get_changed_files` in the beginning of the parsing, and use the result on each object.